tailwind.config.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-dupe-keys */
+const { green } = require('@mui/material/colors');

```

Comment:
Why?

Reply:
Removed,I was trying things and I must have forgotten to delete it